tasks/libs/common/junit_upload_core.py

```diff
@@ -29,14 +29,20 @@
 E2E_INTERNAL_ERROR_STRING = "E2E INTERNAL ERROR"
 CODEOWNERS_ORG_PREFIX = "@DataDog/"
 REPO_NAME_PREFIX = "github.com/DataDog/datadog-agent/"
-if platform.system() == "Windows":
-    DATADOG_CI_COMMAND = [r"c:\devtools\datadog-ci\datadog-ci", "junit", "upload"]
-else:
-    DATADOG_CI_COMMAND = [which("datadog-ci"), "junit", "upload"]
 JOB_ENV_FILE_NAME = "job_env.txt"
 TAGS_FILE_NAME = "tags.txt"
 
 
+def get_datadog_ci_command():
+    if platform.system() == "Windows":
+        return r"c:\devtools\datadog-ci\datadog-ci"
+    else:
+        path_datadog_ci = which("datadog-ci")
+        if path_datadog_ci is None:
+            raise FileNotFoundError("datadog-ci command not found")
```

Comment:
Should we do the same thing for Windows?

Reply:
The way it was coded legacy implied that we were just hardcoding the path for Windows.

I have an intuition that this means in the installation that `datadog-ci` was not added to the PATH. That's why I left it as is, I just wanted to have a clear error, but I don't have the time to check how the installation is made !

Reply:
It was [added to PATH](https://github.com/DataDog/datadog-agent-buildimages/blob/main/windows/helpers/phase3/install_ci_uploader.ps1#L22) though.
I don't remember exactly if I put the exact path to be sure I won't face issues, because testing on windows is so tedious

Reply:
Okk let's try it then !

Reply:
It was incorrectly added, I'll do a PR